@JRC/bootstrap.m

```diff
@@ -322,6 +331,38 @@ function bootstrap(obj, varargin)
         end
     end
 end
+
+function [trialFile,psthTimeLimits] = getTrialFile(workingdir, ask)
+    if nargin < 2
+        ask = 0;
+    end
+
+    trialFile = trialFileInDir(workingdir);
+
+    if ~isempty(trialFile) && ask % found a trial file in working directory; confirm
+        dlgAns = questdlg(sprintf('Found trial file ''%s''. Use it?', trialFile));
+        if strcmp(dlgAns, 'Yes')
+            return;
```

Comment:
Good point. I didn't test that bit because I skip dialog pop ups (I set `ask` to false).
Setting `psthTimeLimits` in the `else` on line 287 is fine. It's overwritten by the settings in the .meta file anyway. This may be improved later.